src/goto-symex/renaming_level.cpp

```diff
@@ -263,6 +263,10 @@ bool check_renaming(const exprt &expr)
     if(to_ssa_expr(expr).get_original_expr().type() != type)
       return true;
   }
+  else if(expr.id() == ID_nil)
+  {
```

Comment:
Is that still needed after the change in `goto_symex_state.cpp`?

Reply:
This is just the sanity check (when using `--validate-ssa-equation`) - hopefully it never fails...